audbcards/core/dataset.py

```diff
@@ -64,6 +94,33 @@ def __init__(
                 audeer.path(self.cache_root, name, other_version)
             )
 
+        # save instance to cache
+        object_cache = audeer.path(
+            self.cache_root,
+            'dataset',
+            name,
+            version,
+            'dataset.pkl',
+        )
+        self._save_pickled(object_cache)
+
+    @staticmethod
+    def _load_pickled(path: str):
+        r"""Load pickled object instance."""
+        if not os.path.exists(path):
+            raise FileNotFoundError()
+
```

Comment:
This error will rarely happen and it will be very hard to test for it as you need to remove the cache file after it was detected to exist, but before reading from it starts.
Also `open()` will most likely raise the same error if the file was really removed, so I would propose to remove it
```suggestion
```

Reply:
This does not exist any more.